WORKING_GROUPS.md

```diff
@@ -287,6 +288,18 @@ Its responsibilities are:
 * Creating and operating a process for documentation review that produces
   quality documentation and avoids impeding the progress of Core work.
 
+### [Testing](https://github.com/nodejs/testing)
+
+The Node.js Testing Working Group's purpose is to extend and improve testing of
+the Node.js source code.
+
+It's responsibilities are:
+
+* Coordinate an overall strategy for improving testing
+* Document guidelines around tests
+* Work with the Build Working Group to improve continuous integration
+* Improve tooling for testing
+
```

Comment:
`Coordinate` => `Coordinating` etc. per @orangemocha and added a full-stop at the end of each bullet point.